sqlengine/postgres_engine_test.go

```diff
@@ -178,6 +178,12 @@ var _ = Describe("PostgresEngine", func() {
 		dropTestUser(template1ConnectionString, masterUsername)
 	})
 
+	It("can construct JDBC URI with ssl option", func() {
+		jdbcuri := postgresEngine.JDBCURI(address, port, dbname, masterUsername, masterPassword)
+		Expect(jdbcuri).ToNot(ContainSubstring("&ssl=true"))
```

Comment:
This isn't really testing the functionality. This test would pass with the code unchanged.

Can we not set `postgresEngine.requireSSL` to true in this test, and then test the desired behaviour. We could even cover both cases - enabled and disabled.